docs/source/libvirt.rst

```diff
@@ -20,48 +20,51 @@ Within Linchpin, the :term:`libvirt_node` :term:`resource_definition` has more
+| networks           | false | list     | xml: networks | Assigns the domain to a network                        |
+|                    |       |          |               | by name. Each device is named                          |
+|                    |       |          | * name (req)  | with an incremented value (eth0)                       |
+|                    |       |          | * ip          |                                                        |
+|                    |       |          | * mac         | .. note:: Network must exist                           |
+|                    |       |          |               |                                                        |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| image_src          | false | string   | virt-install  |                     |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| network_bridge     | false | string   | virt-install  | virbr0              |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| ssh_key            | false | string   | role          | resource_group_name |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| remote_user        | false | string   | role          | ansible_user_id     |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| cloud_config       | false | list     | role          | http://cloudinit.readthedocs.io                        |
+|                    |       |          |               | is used here                                           |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| additional_storage | false | string   | role          | 1G                  |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| uri                | false | string   | module: uri   | qemu:///system      |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| count              | false | string   | N/A           |                     |                                  |
++--------------------+-------+----------+---------------+---------------------+----------------------------------+
+| delete_on_destroy  | false | boolean  | N/A           | False               | If true, libvirt destroy will    |
```

Comment:
@14rcole I think delete_on_destroy should also be on libvirt_network documentation, shouldn't it?